ui/src/containers/pipelines/ShowPipeline.js

```diff
@@ -74,6 +74,12 @@ class ShowPipeline extends Component {
             apiPath={`/pipelines/${pipeline.uuid}`}
             buttons={
               <>
+                <a
+                  href={`/pipelines/${pipeline.uuid}/edit-payload`}
+                  className="btn btn-secondary text-white ms-2"
```

Comment:
My apologies for suggesting `btn-secondary` here, it looks a bit weird and hard to read. What about using the same class as for the other buttons, i.e., `btn-light`?

Reply:
Assuming, you would like to highlight `Edit in PipelineDesigner`, I used `btn-light btn-secondary` and removed `text-white`. That way, we have a similar look for all buttons apart from `Edit in PipelineDesigner`.

Reply:
👍 Looks good, thanks a lot!